README.md

```diff
@@ -144,27 +160,33 @@ Release draft template can be modified by editing `.github/release-drafter.yml`
 It demonstrates how draft should be presented.
 
 ### Testing
+
 [Pytest](https://github.com/pytest-dev/pytest/) is our main test runner.
 
 ### Linting
+
 Code is double-checked during development process. One at commit phase, and the other at CI process.
 
 [pre-commit](https://pre-commit.com/) is help us to check at commit time. By executing installation command `pre-commit install`,
 It automatically adds pre commit hooks. Types of hook are managed using `.pre-commit-config.yaml`.
 
 ### Coverage
+
 Coverage of test functions is one of important metrics which decides code quality.
-`ci.yml` workflow runs unittest and reports coverage report on pull request comment.
-We use [orgoros's github action](https://github.com/orgoro/coverage) as our coverage component of our CI workflow.
+GitHub actions `ci.yml` workflow's unittest job is control coverage report.
+
+We suggest to install Codecov GitHub App which can manage coverage of repository.
```

Comment:
This change should also be reverted as well. We do not use Codecov.

Reply:
Oh, I missed it! Thank you for your confirmation.